tst/testinstall/combinat.tst

```diff
@@ -487,6 +487,35 @@ gap> for s in [[], [5], [1,2,3,4], [2,5,7], ["a","b","c","d","e"], [3..13]] do
 >      elif Set(pn) <> Set(list) then
 >        Error( "different elements" );
 >      fi;
+>      for k in [1 .. Size(s)] do
```

Comment:
If you do this, you need to add `k` to the `#@local` at the start of the file.

Reply:
Thanks for identifying the source of the error for me!